molecule/latest/tests/test_latest.py

```diff
@@ -0,0 +1,30 @@
+"""Module containing the tests for the latest scenario."""
+
+# Standard Python Libraries
+import os
+
+# Third-Party Libraries
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ["MOLECULE_INVENTORY_FILE"]
+).get_hosts("all")
+
+
+@pytest.mark.parametrize("pkg", ["expect", "jq", "nessus"])
+def test_packages(host, pkg):
+    """Test that the appropriate packages were installed."""
+    assert host.package(pkg).is_installed
+
+
+def test_nessus_version(host):
+    """Check that the correct version of Nessus was installed."""
+    cmd = host.run("/opt/nessus/sbin/nessusd --version")
+    assert " 10.3.0 " in cmd.stdout
+
+
+@pytest.mark.parametrize("f", ["/tmp/Nessus-10.3.0-debian9_amd64.deb"])
```

Comment:
It's not a big deal since it's only in two places, but what about making a variable to hold the version number (`10.3.0`) so that we only have to change it in one place when they release a new version?  Is there a way to grab that version number from the Ansible variable in `molecule/latest/converge.yml`?

Reply:
Please see commit 7dc9538 and let me know what you think.

I'd like to combine the two sets of very similar tests into a single file that is shared between the two `molecule` scenarios, but right now I don't see a clean way to do that.  Please let me know if you have any thoughts @dav3r and @mcdonnnj.

Reply:
https://github.com/cisagov/ansible-role-nessus/commit/0b62bee1af9bab4ea456b9cc820e48bddd9d60f6 looks great to me.  I don't have any brilliant ideas to combine those two tests - maybe just create an issue for that and move on?

Reply:
Please see #22 and commit 6cb12af.